deisctl/cmd/cmd.go

```diff
@@ -184,13 +184,19 @@ func startDefaultServices(b backend.Backend, wg *sync.WaitGroup, outchan chan st
 	wg.Wait()
 
 	// optimization: start all remaining services in the background
+	// cache is a special case because it must start before registry
+	b.Start([]string{"cache"}, &_wg, _outchan, _errchan)
```

Comment:
I'm confused. So we first start cache (this line), then we start the entire platform (line 191), then we try starting the cache again (line 197), then we start the database, registry and the controller (line 199)?